editor/editor_node.h

```diff
@@ -323,7 +323,7 @@ class EditorNode : public Node {
 	Control *vp_base = nullptr;
 
 	EditorTitleBar *menu_hb = nullptr;
-	Control *main_control = nullptr;
+	Control *main_screen_vbox = nullptr;
```

Comment:
Is this possible to do? If so, the method to get it could also have its return type changed.
```suggestion
	VBoxContainer *main_screen_vbox = nullptr;
```

Reply:
Oh, right, can fix that as well.